### PR TITLE
Adding 2 new quant tutorials

### PR DESCRIPTION
**Pull request by jlin27**
*  (experimental) Quantized Transfer Learning for Computer Vision Tutorial
https://github.com/pytorch/tutorials/blob/jlin27-quant-tutorials/intermediate_source/quantized_transfer_learning_tutorial.rst

* (experimental) Dynamic Quantization on HuggingFace BERT model
https://github.com/pytorch/tutorials/blob/jlin27-quant-tutorials/intermediate_source/dynamic_quantization_bert_tutorial.rst

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -96,7 +96,7 @@ download:
 	# Download dataset for advanced_source/static_quantization_tutorial.py
 	wget -N https://s3.amazonaws.com/pytorch-tutorial-assets/imagenet_1k.zip -P $(DATADIR)
 	unzip -q -o $(DATADIR)/imagenet_1k.zip -d advanced_source/data/
-
+
```
Comment (z-a-f) [inline Makefile]: nit: Whitespaces

File: intermediate_source/quantized_transfer_learning_tutorial.rst [hunk 2]
```diff
@@ -0,0 +1,510 @@
...
+num_ftrs = model.fc.in_features
+
+# Step 1
+model.train()
+model.fuse_model()
+# Step 2
+model_ft = create_combined_model(model)
+model_ft[0].qconfig = torch.quantization.default_qat_qconfig  # Use default QAT configuration
+# Step 3
+model_ft = torch.quantization.prepare_qat(model_ft, inplace=True)
```
Comment (z-a-f) [inline intermediate_source/quantized_transfer_learning_tutorial.rst]:
```
.. code:: python

    # notice `quantize=False`
    model = models.resnet18(pretrained=True, progress=True, quantize=False)
    num_ftrs = model.fc.in_features

    # Step 1
    model.train()
    model.fuse_model()
    # Step 2
    model_ft = create_combined_model(model)
    model_ft[0].qconfig = torch.quantization.default_qat_qconfig  # Use default QAT configuration
    # Step 3
    model_ft = torch.quantization.prepare_qat(model_ft, inplace=True)
```

File: intermediate_source/quantized_transfer_learning_tutorial.rst [hunk 3]
```diff
@@ -0,0 +1,510 @@
...
+
+To convert the fine tuned model into a quantized model you can call the
+``torch.quantization.convert`` function (in our case only
+the feature extractor is quantized).
+
+.. note:: Because of the random initialization your results might differ from
+   the results shown in this tutorial.
+
+
+# notice `quantize=False`
```
Comment (raghuramank100) [inline intermediate_source/quantized_transfer_learning_tutorial.rst]: This part from 449-451 needs to be replaced with actual code:
```python
# Step 1 
model.train() 
model.fuse_model() 
# Step 2 
model_ft = create_combined_model(model)
model_ft[0].qconfig = torch.quantization.default_qat_qconfig 
# Use default QAT configuration 
# Step 3 
model_ft = torch.quantization.prepare_qat(model_ft, inplace=True)
```

File: intermediate_source/quantized_transfer_learning_tutorial.rst [hunk 4]
```diff
@@ -0,0 +1,510 @@
...
+manually.
+
+Step (2) is performed by the ``create_combined_model`` function
+used in the previous section.
+
+Step (3) is achieved by using ``torch.quantization.prepare_qat``, which
+inserts fake-quantization modules.
+
+
+As step (4), you can start "finetuning" the model, and after that convert
```
Comment (raghuramank100) [inline intermediate_source/quantized_transfer_learning_tutorial.rst]: Minor comment, suggest replacing with:
Once steps  1-3 are done, we can start fine tuning the model (step 4), followed by converting the model into a quantized. (step 5).

File: intermediate_source/dynamic_quantization_bert_tutorial.rst [hunk 5]
```diff
@@ -0,0 +1,550 @@
...
+<https://gist.github.com/W4ngatang/60c2bdb54d156a41194446737ce03e2e>`_
+and unpack it to a directory ``glue_data``.
+
+
+.. code:: shell
+
+   python download_glue_data.py --data_dir='glue_data' --tasks='MRPC'
+
+
+1.4 Learn about helper functions
```
Comment (raghuramank100) [inline intermediate_source/dynamic_quantization_bert_tutorial.rst]: Suggest merging this section with 1.2 as we are explaining what we are doing in 1.2 here, suggested change is shown above.

File: intermediate_source/dynamic_quantization_bert_tutorial.rst [hunk 6]
```diff
@@ -0,0 +1,550 @@
...
+    logger = logging.getLogger(__name__)
+    logging.basicConfig(format = '%(asctime)s - %(levelname)s - %(name)s -   %(message)s',
+                        datefmt = '%m/%d/%Y %H:%M:%S',
+                        level = logging.WARN)
+
+    logging.getLogger("transformers.modeling_utils").setLevel(
+       logging.WARN)  # Reduce logging
+
+    print(torch.__version__)
+
```
Comment (raghuramank100) [inline intermediate_source/dynamic_quantization_bert_tutorial.rst]: The helper functions are built-in in transformers library. We mainly use
the following helper functions: one for converting the text examples
into the feature vectors; The other one for measuring the F1 score of
the predicted result.

The `glue_convert_examples_to_features <https://github.com/huggingface/transformers/blob/master/transformers/data/processors/glue.py>`_ function converts the texts into input features:

-  Tokenize the input sequences;
-  Insert [CLS] in the beginning;
-  Insert [SEP] between the first sentence and the second sentence, and
   in the end;
-  Generate token type ids to indicate whether a token belongs to the
   first sequence or the second sequence.

The `glue_compute_metrics` function computes the  `F1 score <https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html>`_
can be interpreted as a weighted average of the precision and recall,
where an F1 score reaches its best value at 1 and worst score at 0. The
relative contribution of precision and recall to the F1 score are equal.

-  The equation for the F1 score is:
.. math:: F1 = 2 * (\text{precision} * \text{recall}) / (\text{precision} + \text{recall})

Comment (raghuramank100) [reply intermediate_source/dynamic_quantization_bert_tutorial.rst]: @jianyuh : Does this look ok?

**Pull request merged**